libs/tup-components/src/tickets/TicketDetailModal/TicketReplyForm.tsx

```diff
@@ -120,30 +124,24 @@ export const TicketReplyForm: React.FC<{
                     id="status"
                     checked
                     disabled
-                  />{' '}
+                  />
                   My issue has been resolved
                 </div>
               )}
-              {ticketData?.Status === 'resolved' ? (
+
+              {isResolved ? (
                 <em>*Replying will reopen this ticket</em>
               ) : (
                 <em>
                   This helps us determine which users still need assistance
                 </em>
               )}
             </div>
-            {/* <FormikCheck 
-              name="status"
-              label="Ticket Status"
-              description="This helps us determine which users still need assistance"
-            /> */}
             <FormGroup className="ticket-reply-submission">
               <Button
                 attr="submit"
                 type="primary"
-                disabled={
-                  !isValid || isSubmitting || isLoading || isError || !dirty
-                }
+                disabled={values.text.length === 0 && !isChecked}
```

Comment:
**Variable Opportunity.** As discussed, consider variable for this reused `values.text.length === 0` reference.